brave/brave-ios
# Fix #8768: Add Logging to Track Secure Content State

## Summary of Changes

Customizing the Logger and adding secure state logging


This pull request fixes brave/brave-browser#36281

This Pr will be also opened in brave-core side

## Submitter Checklist:


## Test Plan:


- Run the application 
- Go .. dot menu and settings
- Find Secure Content Debug at the bottom
- Check the logs or share

## Screenshots:



https://github.com/brave/brave-ios/assets/6643505/0b3259ca-f3d3-4565-b686-1ac377e5ddaa


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
